tobler/area_weighted/area_interpolate_dask.py

```diff
@@ -0,0 +1,261 @@
+'''
+Area Weighted Interpolation, out-of-core and parallel through Dask
+'''
+
+import pandas
+import geopandas
+import dask_geopandas
+import numpy as np
+from dask.base import tokenize
+from dask.highlevelgraph import HighLevelGraph
```

Comment:
we need to move these into a try/except inside the function (like with [h3fy](https://github.com/pysal/tobler/blob/df0cbc6821cdf8c84c7f8b3dcfd1d60eebbb252e/tobler/util/util.py#L183)), otherwise they will force dask to be installed on import

Reply:
(Levi also has a more clever way of [doing this with decorators](https://github.com/pysal/spreg/blob/main/spreg/optional_imports.ipynb))